InscryptionAPI/Saves/ModdedSaveManager.cs

```diff
@@ -0,0 +1,80 @@
+using DiskCardGame;
+using HarmonyLib;
+using System.Reflection;
+
+namespace InscryptionAPI.Saves
+{
+    [HarmonyPatch]
+    public static class ModdedSaveManager
+    {
+        private static readonly string saveFilePath = Path.Combine(Assembly.GetExecutingAssembly().Location, "..", "..", "ModdedSaveFile.gwsave");
```

Comment:
Is this meant to point to `BepInEx/ModdedSaveFile.gwsave`? If so, just use BepInEx
Paths to get the path to the BepInEx directory

Reply:
I switched this to use the bepinex.paths helper